### PR TITLE
Building My Capstone Project

### PR DESCRIPTION
**Pull request by gisachris**
👋 Hi there! client TSE 😉 

I present 🎁  a pull request on building my module one capstone project and below is a small description 📜 .And as always I look forward to hearing 👂 from you.

## Summary

This pull request consists of the entire capstone build! Here's a quick summary of what is in this pull request:

- Mobile and Desktop Version Website Fully Responsive with  `(min-width:768px)` & `(max-width:768px)` breakpoints.
- Incorporate javascript for better user experience 🤸‍♂️  and navigation 🧑‍🔧  of the website.
- Incorporate javascript on adding a `clients` section in the page 🥇 
- Personalise the website to my own general idea. 🧠 
- Use bootstrap framework ✉️ and sass preprocessor 🎡  to build the website. 

## Conclusion

Overall, I have put in a great deal of energy ⚡  on this project and pull request! Please let me know if you have any questions or concerns about my work. I'm hoping the website is good enough to have your acceptance 😄  and get merged into the main branch soon.

## Communication
if possible after review please leave your slack handle so I can ask you more questions 🙇.thank you.

👍👍👍

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
+        </section>
+        <section class="service-5">
+          <img src="assets/images/patterns/atom-e.png" alt="service 1 image">
+          <h2 class="service-title">Networking</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
+        </section>
+        </section>
+        <button class="button-services">
+          <a href="index.html" class="but-text">JOIN THE FIGHT FOR JUSTICE!</a>
```
Comment (tufoinnkuo10) [inline index.html:112]: - [ ] please kindly give the cards a slightly darker background so it match the original design. also, the JOIN the fight for justice should be underlined like in the design. remember you are only free to personalize content but the design should be followed.
yours
Screenshot (1010)_LI
original
Screenshot (1011)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        </section>
+        <section class="service-4">
+          <img src="assets/images/patterns/blackboard-e.png" alt="service 1 image">
+          <h2 class="service-title">Mediation</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
+        </section>
+        <section class="service-5">
+          <img src="assets/images/patterns/atom-e.png" alt="service 1 image">
+          <h2 class="service-title">Networking</h2>
+          <p class="service-par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit natus</p>
```
Comment (tufoinnkuo10) [inline index.html:108]: - [ ] please increase the heights of the cards a little so as to make them more visible and also so it looks like the original design. nice job so far.
your design
Screenshot (1015)_LI
original
Screenshot (1000)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    </header>
+    <main class="container-fluid container-default homepage">
+      <section class="fade-image">
+        <span class="img"></span>
+    </section>
+    <section class="rest-homepage">
+      <section class="home-text-section">
+        <h1 class="home-text-1">Hello There!</h1>
+        <h2 class="home-text-2">Welcome to lawyers for justice</h2>
+        <p class="paragraph-home-1">Visit out humble website below.<br>We would love to here from you and what we can do for you.Advocating for positive change with people over 80 countries</p>
```
Comment (tufoinnkuo10) [inline index.html:55]: - [ ] please this text is in a block form with white borders . kindly follow the design but feel free to personalize the content.
- [ ] make the date bolder and add text under it
yours
Screenshot (1009)_LI
main design

Screenshot (1008)

File: assets/dist/js/toptalent.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    if (button.classList.contains('expand')) {
+      button.textContent = 'More';
+    } else if (button.classList.contains('extend')) {
+      button.textContent = 'Less';
+    }
+
+    const lawyerArray = document.querySelectorAll('.talent1container');
+    const expandables = Array.from(lawyerArray).slice(2);
+    for (let j = 0; j < expandables.length; j++ /* eslint-disable-line no-plusplus */) {
+      // expandables[j].classList.add('expandables');
```
Comment (tufoinnkuo10) [inline assets/dist/js/toptalent.js]: - [ ] please remove the comment to keep your code clean

File: assets/dist/js/navigation.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+// selecting the navigation
+const navigation = document.querySelector('.nav-together');
+// const navBar = document.querySelector('.nav-bar');
```
Comment (tufoinnkuo10) [inline assets/dist/js/navigation.js]: - [ ] kindly remove the commented code for clean code practice

File: index.html [hunk 6]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>lawyers for justice website</title>
+  <!-- <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous"> -->
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] kindly remove comments for clean code practices

File: assets/dist/js/toptalent.js [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+  // start to lawyers section
+
+  const lawyersContainer = document.createElement('div');
+  lawyersContainer.classList.add('tcontainer');
+  talentHolder.append(lawyersContainer);
+
+  const talents = [
+    {
+      img: 'assets/images/act/lawyerp1.jpg',
+      header: 'Consultation',
```
Comment (tufoinnkuo10) [inline assets/dist/js/toptalent.js:35]: - [ ] please ensure that Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description like in the original design
yours
Screenshot (1006)
behance design
Screenshot (1012)

File: about.html [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+         <img src="assets/images/google.png" alt="a company we have worked with">
+         <img src="assets/images/naver.png" alt="a company we have worked with">
+         <img src="assets/images/microsoft.png" alt="a company we have worked with">
+         <img src="assets/images/airbnb.png" alt="a company we have worked with">
+       </section>
+      </section>
+      <section class="whitesection">
+        <section class="comImg">
+          <img src="assets/images/creativecommons.png" alt="a logo for copyright">
+          <h2 class="copyrightheader">Creative<br>Commons Grobal</h2>
```
Comment (tufoinnkuo10) [inline about.html:96]: - [ ] please your desktop home page is missing a the partners section and footer, it includes a logo to the left and some text to the right. please kindly add that like in the original design.
original design
Screenshot (1007)

File: about.html [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+      <section class="past">
+        <article class="blackheader">
+          <h2 class="main-text">Past Trials</h2>
+          <img src="assets/images/patterns/underline.png" alt="underline for text">
+        </article>
+        <p class="missionstatement">take a look at some of our past trials and how we handled them with leading experts in law.</p>
+
+        <section class="past-images">
+          <figure class="img1"><img src="assets/images/haiti.jpg"  class="img-deep" alt="haiti images"></figure>
+          <figure class="img"><img src="assets/images/ukraine.jpg" class="img-deep" alt="haiti images"></figure>
```
Comment (tufoinnkuo10) [inline about.html:74]: - [optional] you can reduce the image sizes. they kind of look too big compared to the main design.

File: about.html [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+        </div>
+        <img src="assets/icons/ham-icon.svg" class="ham-icon" alt="menu">
+    </header>
+    <main class="container-fluid container-default aboutpage">
+      <section class="fade-image">
+        <span class="img"></span>
+    </section>
+    <section class="rest-aboutpage">
+      <section class="about-text-section">
+        <h2 class="about-text-2">About Us</h2>
```
Comment (tufoinnkuo10) [inline about.html:51]: - [ ] you can use the title you used in the home page and please ensure it is centralized. 
- [ ] please use the normal color,(\#EC5242) as the about page heading has no design
yours 
Screenshot (1013)_LI
original
Screenshot (1014)

File: README.md [hunk 11]
```diff
@@ -1,2 +1,196 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my fellow coding partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/gisachris/week1_portfolio/blob/new-features/MIT.md) licensed.
```
Comment (tufoinnkuo10) [inline README.md]: - [ ] Nice job so far. please you MIT link is broken. kindly fix it
Screenshot (1004)

File: README.md [hunk 12]
```diff
@@ -1,2 +1,196 @@
...
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project consider viewing my other profiles and sending job inquiries. thank you
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```
Comment (tufoinnkuo10) [inline README.md:181]: - [ ] please kindly give the deserved and  appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in this section

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @gisachris 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ great design

video presentation
Great job with your presentation. you were straight to the point. please always give a good reason why you created the project and how you think it can be of help.

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 12 above)
Comment (gisachris) [reply README.md:181]: ## As per my previous CR 🔍 i was requested ♻️ to make some changes :
Below are the changes ♻️  I have Made to fix the issues 👨‍💻 :  
- fix header text to a block form with white borders and change the date to a bolder look🥇.
-  remove unnecessary comments to keep your code clean. 🟢 
- Resize the featured speaker section(top clients) to match the design.🟢
- Add a footer section to the homepage.🟢
## [optional changes that were made]
-reduce the image sizes on the About page. to fit the main design.🟢
-fix about page title as recommended.🟢
.fix MIT link.🟢
-give the deserved and appropriate credit to Cindy Shin the author of the original design.🟢

as per my last commit 🟢 to the pull request i have fixed 🧑‍🔧 all the issues above. 🏅
I hope to hear from you my CR 🤺 soon!.

**Review by braincee — APPROVED** [review]
Hi @gisachris 👋 , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**